### ISSUE TITLE
youtube

### ISSUE THREAD

**Issue by EllaSTV** [l-atelier-des-chercheurs/Les-Cahiers-du-Studio#30]
Serait-il possible de cliquer sur un lien youtube et d'être directement redirigé vers internet ? plutôt que de faire un copié collé ?

**Comment by EllaSTV** [comment]
Dans le projet rencontre les images ne sont toujours pas visible, alors qu'elles sont bien dans le documents les cahiers

**Comment by louis-ev** [comment]
Dans un bloc texte du coup ?

**Comment by EllaSTV** [comment]
oui, j'ai mis un lien dans un bloc texte et ça ne génère pas de lien internet
